src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -628,6 +628,7 @@ protected void Accepting()
                         _transportMapping[send.Recipient.LocalAddressToUse], _settings, writing: true,
                         handleOption: null, refuseUid: refuseUid), uid: null, refuseUid: refuseUid);
 
+                // pattern match won't throw a NullReferenceException if one is returned by WritableEndpointWithPolicyFor
```

Comment:
Added a comment here just to clarify - `_endpoints.WritableEndpointWithPolicyFor` can return `null`, but `PatterMatch` will not throw and will route the call to the `Default` match below, which is the correct behavior.